karton/misp_pusher/misp_pusher.py

```diff
@@ -39,6 +41,15 @@ def __init__(self, *args, **kwargs) -> None:
         if not self.config.get("misp", "key"):
             raise RuntimeError("Misp config section is missing the key parameter")
 
+        self.cluster_mapping = None
```

Comment:
nitpick, but maybe consider `{}` instead of `None` (such that it's always a dict - looks like it's already handled correctly later)

Reply:
```suggestion
        self.cluster_mapping = {}
```